src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqBwaSpark.java

```diff
@@ -137,13 +125,14 @@ protected void runTool(final JavaSparkContext ctx) {
         }
         final ReadsSparkSource readsSource = new ReadsSparkSource(ctx, readArguments.getReadValidationStringency());
         final PipelineOptions options = getAuthenticatedGCSOptions();
-        try (final PSBwaAlignerSpark aligner = new PSBwaAlignerSpark(ctx, bwaArgs)) {
-            boolean bPairedSuccess = alignBam(inputPaired, aligner, true, ctx, options, readsSource);
-            boolean bUnpairedSuccess = alignBam(inputUnpaired, aligner, false, ctx, options, readsSource);
-            if (!bPairedSuccess && !bUnpairedSuccess) {
-                throw new UserException.BadInput("No reads were loaded. Ensure --pairedInput and/or --unpairedInput are set and valid.");
-            }
+
+        final PSBwaAlignerSpark aligner = new PSBwaAlignerSpark(ctx, bwaArgs);
+        boolean bPairedSuccess = alignBam(inputPaired, aligner, true, ctx, options, readsSource);
+        boolean bUnpairedSuccess = alignBam(inputUnpaired, aligner, false, ctx, options, readsSource);
+        if (!bPairedSuccess && !bUnpairedSuccess) {
+            throw new UserException.BadInput("No reads were loaded. Ensure --pairedInput and/or --unpairedInput are set and valid.");
         }
+        aligner.close();
```

Comment:
Do you not care about having this get closed in case of exception any more? Just wondering why you got rid of the try block.